pages/vi/firststeps.md

```diff
@@ -114,6 +114,8 @@ Make sure that you've linked to your github.io and pull request in the [Gitter c
 
 * 1 Pull request made
 
+Check your progress [here](https://treehouses.io/#!./pages/vi/trackprogress.md)
```

Comment:
please do not call the whole url if shorter options like relative ones or just filenames are working